components/developer/gcc-11/Makefile

```diff
@@ -34,6 +35,8 @@ GMP_ARCHIVE_HASH=	sha256:eae9326beb4158c386e39a356818031bd28f3124cf915f8c5b1dc4c
 
 include $(WS_MAKE_RULES)/gcc-component.mk
 
+COMPONENT_BUILD_ARGS=	-j4
```

Comment:
Please do not do this.  If it is safe to use parallel build for this component then please set `USE_PARALLEL_BUILD = yes` before the `shared-macros.mk` include.

Reply:
removed